2fa-api/handlers/create.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../db");
+
+const create = function (req, res) {
+  try {
+    const { email } = req.body;
+    const { password } = req.body;
+    const { phone } = req.body;
+
+    db.connection.query(
+      `INSERT INTO Users (email, password, phone) VALUES ('${email}', '${password}', '${phone}')`,
```

Comment:
you're doing a raw sql query. can you foresee anything wrong with that?

Reply:
SQL Injection attack

Reply:
give me an example

Reply:
someone hits this endpoint with subqueries in place of email, password, etc. that allow them to read sensitive information, overwrite data, or even delete data / tables.